metadata-integration/java/spark-lineage/src/main/java/datahub/spark/consumer/impl/McpEmitter.java

```diff
@@ -92,6 +110,38 @@ public McpEmitter(Config datahubConf) {
               .server(gmsUrl).token(token)
               .disableSslVerification(disableSslVerification).build());
           
+          break;
+      case "kafka":
+          String kafkaBootstrap;
+          if (datahubConf.hasPath(KAFKA_BOOTSTRAP)) {
+             kafkaBootstrap = datahubConf.getString(KAFKA_BOOTSTRAP);
+          } else {
+              log.error("Kafka Emitter requires Kafka bootstrap config: {}", KAFKA_BOOTSTRAP);
+              break;
+          }
+          String schemaRegistryUrl;
+          if (datahubConf.hasPath(KAFKA_SCHEMA_REGISTRY_URL)) {
+              schemaRegistryUrl = datahubConf.getString(KAFKA_SCHEMA_REGISTRY_URL);
+          } else {
+              log.error("Kafka Emitter requires Schema Registry URL config: {}", KAFKA_SCHEMA_REGISTRY_URL);
+              break;
```

Comment:
Should we throw an exception here? Maybe IllegalArgumentException or the like? Breaking may not do what we want here

Reply:
I've updated the PR to throw an IllegalArgumentException instead of simply breaking.

Reply:
@jjoyce0510 Could you take a look at this again?